src/main/java/backend/Logic.java

```diff
@@ -127,9 +127,9 @@ public void refresh() {
         });
     }
 
-    public CompletableFuture<Boolean> getIssueMetadata(String repoId, List<Integer> issues) {
+    public CompletableFuture<Boolean> getIssueMetadata(String repoId, Map<Integer, String> issues) {
```

Comment:
This (issue id, ETag) map is used a lot throughout. Could we make this a class? Alternatively we could pass a collection of (copied) `TurboIssue`s around, but I'm not sure if the copy could still be affected by mutation of the originals.